### DISCUSSION TITLE
Parts list

### DISCUSSION THREAD

**Discussion by johnkinsella1** [Zer0-bit/gaggiuino#316] [Q&A] [answered]
Just purchased mk1 classic and really looking forward to doing this mod! Perhaps I'm missing something but I can't seem to find a parts list? 

Secondly then is the black pill rout much more difficult, or is it easy enough to upgrade arduino method after?

Thanks in advance!

**Comment by banoz** [comment, marked as answer]
> Just purchased mk1 classic and really looking forward to doing this mod! Perhaps I'm missing something but I can't seem to find a parts list?
> 
> Secondly then is the black pill rout much more difficult, or is it easy enough to upgrade arduino method after?

All the documentation moved to https://gaggiuino.github.io
